mypy/test/testcheck.py

```diff
@@ -278,22 +278,17 @@ def parse_module(self, program_text: str, incremental: int = 0) -> List[Tuple[st
 
     def parse_options(self, program_text: str, testcase: DataDrivenTestCase) -> Options:
         options = Options()
-        flags = re.search('# options: (.*)$', program_text, flags=re.MULTILINE)
-        version_flag = re.search('# pyversion: (.*)$', program_text, flags=re.MULTILINE)
-        platform_flag = re.search('# platform: (.*)$', program_text, flags=re.MULTILINE)
+        flags = re.search('# flags: (.*)$', program_text, flags=re.MULTILINE)
 
+        flag_list = None
         if flags:
-            options_to_enable = flags.group(1).split()
-            for opt in options_to_enable:
-                setattr(options, opt, True)
-
-        # Allow custom pyversion comment to override testcase_pyversion
-        if version_flag:
-            options.python_version = parse_version(version_flag.group(1))
+            flag_list = flags.group(1).split()
+            _, options = process_options(flag_list, require_targets=False)
```

Comment:
If we're going to add a TODO, maybe also add a check that raises an error if the user actually tries passing in file names/modules?

Maybe do something like:

```
targets, options = process_options(flag_list, require_targets=False)
if len(targets) != 0:
    # Raise error/TODO message here
```